include/ceed/jit-source/magma/magma-common-tensor.h

```diff
@@ -178,6 +207,23 @@ static __device__ __inline__ void write_V_3d(T *dV, const int compstride, T rV[D
   }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// sum into V of a 3D element from registers rV[][][] to global memory --  for all components of a single dim
+// dV is assumed to point directly to the element (i.e. already offset by elem-stride)
+// register is assumed to be rV[DIM_V][NUM_COMP][rV_SIZE]
+// i_DIM specifies which dimension is being written to in dV
+// rV_SIZE can be different from P (e.g. max(P, Q))
+template <typename T, int Q, int DIM_V, int NUM_COMP, int rV_SIZE, int i_DIM>
+static __device__ __inline__ void sum_V_3d(T *dV, const int compstride, T rV[DIM_V][NUM_COMP][rV_SIZE], const int tx) {
+  if (tx < (Q * Q)) {
+    for (int comp = 0; comp < NUM_COMP; comp++) {
+      for (int j = 0; j < Q; j++) {
+        dV[comp * compstride + j * (Q * Q) + tx] = rV[i_DIM][comp][j];
```

Comment:
Somehow missed getting this one in my review. Need `+=`.

Reply:
Ok, these fixes fixed it all. You are l33t, as the kids say (I think... do they still say that?). Thank you for catching those fixes!

Reply:
Ha, I think copy-paste bugs have this weird thing where they are much easier to spot when you are not the one who did the copy-pasting in the first place. Maybe a quantum phenomenon, where they exist in a state that is both camouflaged and not camouflaged at the same time until observed ;)